ui/ui.go

```diff
@@ -426,10 +424,11 @@ func (this *UI) validateMenuItems(menuItems []dataModels.MenuItem, name string,
 func (this *UI) update() {
 	logger.TraceEnter("ui.update()")
 
+	this.sdNotify(daemon.SdNotifyWatchdog)
+	
 	if this.connectionAttempts > 8 {
 		logger.Info("ui.update() - this.connectionAttempts > 8")
 		this.splashPanel.putOnHold()
-		this.sdNotify(daemon.SdNotifyWatchdog)
```

Comment:
The changes looks good.  Once I merge this in I'll test it out.  The call here on line 432 was added specifically for the case of when the printer is offline.

Reply:
Now, for good measure, it pings the watchdog on every call to update. To make sure that we stay inside the 1/2 timeout timeframe.